tmuxp/workspacebuilder.py

```diff
@@ -384,6 +384,9 @@ def pane_has_same_path(p):
         if all(pane_has_same_path(p) for p in w.panes):
             wconf['start_directory'] = w.panes[0].current_path
 
+        if w.name == '':
+            empty_window_title = True
```

Comment:
```suggestion
        empty_window_title = w.name == ''
```

Assures the variable is always declared

Should fix https://github.com/tmux-python/tmuxp/pull/189/checks?check_run_id=991200510#step:15:372:

```
                wconf['panes'].append(pconf)
>               if empty_window_title:
E               UnboundLocalError: local variable 'empty_window_title' referenced before assignment

tmuxp/workspacebuilder.py:416: UnboundLocalError
```

Reply:
That resolves the unbound variable. I am adding that change.